balanços.html

```diff
@@ -1,88 +1,67 @@
+                <li><a href="suporte.html" class="menu__superior--itens">Suporte de Plantas</a></li>
+                <li><a href="espelho.html" class="menu__superior--itens">Espelhos</a></li>
+                <li><a href="balanços.html" class="menu__superior--itens">Balanços</a></li>
+                <li><a href="bolsas.html" class="menu__superior--itens">Bolsas</a></li>
+                <li><a href="arvore da vida.html" class="menu__superior--itens">Árvores da vida</a></li>
+                <li><a href="quemsoueu.html" class="menu__superior--itens">Quem sou eu</a></li>
+            </ul>
+        </nav>
 
     </div>
 
-    <!--- Quadrado Propaganda ------------------->
-    <div class="quadrado__propaganda">
-        <div class="quadrado__propaganda--txt">
-            <p class="quadrado__propaganda--txt--class">arte</p>
-            <p class="quadrado__propaganda--txt--class box--vinho">única</p>
-            <p class="quadrado__propaganda--txt--class">na sua</p>
-            <p class="quadrado__propaganda--txt--class">casa</p>
-        </div>
-        <div class="quadrado__propaganda--imagem">
-            <img src="./img/logovinhosemfundo5.png" class="quadrado__propaganda--imagem--logo">
+       <!--- Quadrado Propaganda ------------------->
+       <div class="quadrado__propaganda">
+            <img src="./imgdosprodutos/testebanner1.png" class="imagem--banner" alt="Imagem 1">
+            <img src="./imgdosprodutos/testebanner2.png" class="imagem--banner" alt="Imagem 2">
+            <img src="./imgdosprodutos/testebanner3.png" class="imagem--banner" alt="Imagem 3">
         </div>
-    </div>
 
     <div class="divisoria"></div>
 
     <div class="novidades">balanços</div>
 
     <div class="novidades__produto">
         <div class="novidades__produto--01">
-            <img src="./img/balanco.JPG" class="novidades__produto--01--imagem">
-            <div class="novidades__produto--01--info">
-                <p class="novidades__produto--01--txt">Produto 1</p>
-            </div>
-        </div>
-        <div class="novidades__produto--02">
-            <img src="./img/espelho1.png" class="novidades__produto--01--imagem">
-            <div class="novidades__produto--01--info">
-                <p class="novidades__produto--01--txt">Produto 2</p>
-            </div>
-        </div>
-        <div class="novidades__produto--03">
-            <img src="./img/luminaria.JPG" class="novidades__produto--01--imagem">
-            <div class="novidades__produto--01--info">
-                <p class="novidades__produto--01--txt">Produto 3</p>
-            </div>
-        </div>
-    </div>
-
-    <div class="divisoria--menu"></div>
-
-    <div class="novidades__produto">
-        <div class="novidades__produto--01">
-            <img src="./img/balanco.JPG" class="novidades__produto--01--imagem">
+            <img src="./imgdosprodutos/Balanço/balanco1.jpg" class="novidades__produto--01--imagem">
```

Comment:
é bom evitar usar acentos e caracteres especiais no nome dos arquivos e pastas, vc pode usar um formatação "snake case" no nome das pastas tb.

se for plicar essa mudança não esquece de renomear os arquivos e as pastas
```suggestion
            <img src="./img_dos_produtos/balanco/balanco1.jpg" class="novidades__produto--01--imagem">
```

Reply:
Vou fazer isso. Pensei outro dia sobre o assunto.